spec/factories/entry.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
```

Comment:
Right now it makes more sense. We have just three of them and they are very, very small. I'll change it :)